docs/site/migration/auth/overview.md

```diff
@@ -26,3 +26,8 @@ authorization to secure the applications:
   authorize client applications and/or resource owners (i.e. users) to access
   protected API endpoints. The migration path is described in
   [Migrating OAuth2 provider](./oauth2.md).
+
+- [loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control)
```

Comment:
I mentioned this in another comment but don't forget to add the example docs to the [sidebar](https://github.com/strongloop/loopback-next/blob/master/docs/site/sidebars/lb4_sidebar.yml#L772) too 👍

Reply:
😆 forgot the sidebar, added.